Modify Pmin/Pmax code in main.py & Feed in species_list for ThirdBody type kinetics to allow collision limit violation check

hwpang opened the pull request:
### Motivation or Problem
See #1872.

Essentially the original code intended to get reactions to high pressure limit by setting P=1e8 Pa in LiquidReactor, but it couldn't perform the collision limit violation check for PDep type reactions because LiquidReactor doesn't have Prange attribute. (main.py line 677)

```
        try:
            self.Pmin = min([x.Prange[0].value_si if x.Prange else x.P.value_si for x in self.reaction_systems])
            self.Pmax = max([x.Prange[1].value_si if x.Prange else x.P.value_si for x in self.reaction_systems])
        except AttributeError:
            # For LiquidReactor, Pmin and Pmax remain with the default value of `None`
            pass
```

### Description of Changes
Added dummy variable `Prange` in LiquidReactor. With this, Pmin = Pmax = P = 1e8 Pa to get reactions to high pressure limit for LiquidReactor.

### Testing

This is my testing script. The original code gave an AttributeError when trying to access x.Prange. This PR's code can perform the collision limit violation check.

```
from rmgpy.rmg.main import RMG, RMGDatabase
from rmgpy.solver.liquid import LiquidReactor
from rmgpy.kinetics import PDepArrhenius
from rmgpy import settings

rmg = RMG()
rmg.reaction_systems = [LiquidReactor(298,{})]

rmg.Tmin = min([x.Trange[0].value_si if x.Trange else x.T.value_si for x in rmg.reaction_systems])
rmg.Tmax = max([x.Trange[1].value_si if x.Trange else x.T.value_si for x in rmg.reaction_systems])
rmg.Pmin = min([x.Prange[0].value_si if x.Prange else x.P.value_si for x in rmg.reaction_systems])
rmg.Pmax = max([x.Prange[1].value_si if x.Prange else x.P.value_si for x in rmg.reaction_systems])

db = RMGDatabase()
db.load(
    path=settings['database.directory'],
    thermo_libraries=['primaryThermoLibrary'],
    transport_libraries=[],
    reaction_libraries=['NOx2018'],
    seed_mechanisms=[],
    kinetics_families='default',
    kinetics_depositories=['training'],
    depository=False,
)

for rxn in db.kinetics.libraries["NOx2018"].get_library_reactions():
    if isinstance(rxn.kinetics,PDepArrhenius):
        for i, spc in enumerate(rxn.reactants):
            rxn.reactants[i].thermo = db.thermo.get_thermo_data(spc)
        for i, spc in enumerate(rxn.products):
            rxn.products[i].thermo = db.thermo.get_thermo_data(spc)
        PDeprxn = rxn
        break

PDeprxn.check_collision_limit_violation(t_min=rmg.Tmin, t_max=rmg.Tmax,p_min=rmg.Pmin, p_max=rmg.Pmax)
```

hwpang commented:
During investigating above mentioned issue, I realize that the current collision limit violation check doesn't work for ThirdBody type kinetics, because the species_list isn't fed in. The original mainTest before this PR didn't have a ThirdBody type kinetics violator.

alongd commented:
I wonder if we could make this fix "cleaner". It's good that you added the comment next to `self.Prange = None`, because accidentally deleting or changing it is indeed a weakness of this implementation. But it's implementing a solution far away (code-wise) from the problem.
If you agree, I'd recommend directly addressing the issue in [main.py](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/rmgpy/rmg/main.py#L677), so that it correctly deals with any scenario (and won't break if `LiquidReactor` ever changes).
What do you say about something like:

```python
self.Pmin = min([x.Prange[0].value_si if x.hasattr('Prange') and x.Prange
                 else x.P.value_si for x in self.reaction_systems])
```

hwpang commented:
> I wonder if we could make this fix "cleaner". It's good that you added the comment next to `self.Prange = None`, because accidentally deleting or changing it is indeed a weakness of this implementation. But it's implementing a solution far away (code-wise) from the problem.
> If you agree, I'd recommend directly addressing the issue in [main.py](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/rmgpy/rmg/main.py#L677), so that it correctly deals with any scenario (and won't break if `LiquidReactor` ever changes).
> What do you say about something like:
> 
> ```python
> self.Pmin = min([x.Prange[0].value_si if x.hasattr('Prange') and x.Prange
>                  else x.P.value_si for x in self.reaction_systems])
> ```

Thanks for the comment! @alongd I agree that this is a much more elegant way to do it. I modified the code for Pmin/Pmax and removed the now stale try/except block in main.py.
